code/datums/station_traits/negative_traits.dm

```diff
@@ -137,7 +137,7 @@
 /datum/station_trait/bot_languages
 	name = "Bot Language Matrix Malfunction"
 	trait_type = STATION_TRAIT_NEGATIVE
-	weight = 4
+	weight = 2
```

Comment:
This is just simplemobs, not cyborgs I assume. To that end I don't think this is all that annoying? Just leave it as it is.